api/requirements.txt

```diff
@@ -7,3 +7,5 @@ python-dotenv~=1.0.1
 requests~=2.32.3
 python-jose~=3.3.0
 Flask-Cors~=5.0.0
+Pillow~=12.0
+pytest~=6.0
```

Comment:
Is there a reason to pick this old version? on Pypi.org, I see `9.0.2` being available [here](https://pypi.org/project/pytest/).